addOns/selenium/src/main/javahelp/org/zaproxy/zap/extension/selenium/resources/help/contents/intro.html

```diff
@@ -10,8 +10,13 @@ <H1>Selenium</H1>
 	<p>The Selenium add-on provides WebDrivers, for other add-ons, to invoke and remotely
 		control web browsers. It's also bundled the HtmlUnit web browser, an headless Java based
 		web browser.
+		
+	<p>It add a 'Selenium' script type - these scripts are invoked whenever selenium launches a browser,
```

Comment:
Drop "it" and capitalize "Add"? Also for labels and in text should Selenium have a cap as a title/name?

Reply:
If so, changelog too :+1:

Reply:
So this is the help content, not much but its here :)
But it should be "It adds a "...

Reply:
Sorry I dunno what I thought this was when I wrote that summary content 👎 

Thanks ⭐️